### PR TITLE
[draft][IMP] fleet: Add vehicle range to vehicles and models

### PR DESCRIPTION
**Pull request by jugj-odoo**
task-3847484

### CODE REVIEW

File: addons/fleet/models/fleet_vehicle.py [hunk 1]
```diff
@@ -116,6 +116,7 @@ def _get_default_state(self):
         ('today', 'Today'),
     ], compute='_compute_service_activity')
     vehicle_properties = fields.Properties('Properties', definition='model_id.vehicle_properties_definition', copy=True)
+    vehicle_range = fields.Integer(related='model_id.vehicle_range')
```
Comment (Bertrand2) [inline addons/fleet/models/fleet_vehicle.py:119]: the field should be editable for consistency with other model related fields

File: addons/fleet/views/fleet_vehicle_model_views.xml [hunk 2]
```diff
@@ -48,6 +48,10 @@
                             <group string="Engine" invisible="vehicle_type != 'car'">
                                 <group>
                                     <field name="default_fuel_type" required="1"/>
+                                    <label for="vehicle_range"/>
+                                    <div class="o_row" name="vehicle_range">
+                                        <field name="vehicle_range"/><span>km</span>
```
Comment (Bertrand2) [inline addons/fleet/views/fleet_vehicle_model_views.xml:53]: maybe use `o_hr_narrow_field` or `o_hr_narrow_field-size` to make the field less imposing :thinking:
(and maybe it should be done to other fields around that are taking full length when the input will never exceed 10 characters)

**Review by Bertrand2 — CHANGES_REQUESTED** [review]
2 comments, otherwise LGTM! :ok_hand:

File: addons/fleet/views/fleet_vehicle_model_views.xml (hunk 2 above)
Comment (jugj-odoo) [reply addons/fleet/views/fleet_vehicle_model_views.xml:53]: hmm maybe we should do that separately in an improve fleet UX task

**Review by Bertrand2 — APPROVED** [review]

**Comment by Bertrand2** [comment]
robodoo r+

**Pull request closed without merging**